projects/uitpas/docs/test-dataset.md

```diff
@@ -41,30 +41,31 @@ If you want to test ticket-sales without registering your own event, you can use
 
-| UitpasNumber      | Social tariff | Postal code | Coupons | Card status |
-|----------|:-------------:|:-------------:|:-------------:|:-------------:|
-| `0900000095902` | no | 9000 | no | active |
-| `0900000067513` | yes (Gent) | 9090 | no | active |
-| `0900000031618` | expired |  9000 | 1 coupon (Gent) | active |
-| `0900000036112` | yes (Gent) |  9090 | no | active |
-| `0900000002312` | yes (Gent) |  9000 | no | blocked |
-| `0900000058918` | expired |  1000 | no | active |
-| `1000000520006` | no |  9000 | no | active |
-| `0900000093204` | yes (Paspartoe) |  9000 | 2 coupons (Paspartoe) | active |
-| `0900000074519` | yes |  n/a | n/a | unregistered |
-| `0900000075706` | no |  n/a | n/a | unregistered |
+| UitpasNumber    | Social tariff         | Postal code | Coupons                  | Card status  | INSZ-number   | Comment                          |
+| --------------- | :-------------------: | :---------: | :----------------------: | :----------: | :-----------: | :------------------------------: |
+| `0900000095902` | no                    |     9000    |           no             |    active    | `83030201540` | Use this pass for points rewards |
+| `0900000067513` | yes (Gent)            |     9090    |           no             |    active    | `36051380284` | Use this pass for points rewards |
+| `0900000031618` | expired               |     9000    |     1 coupon (Gent)      |    active    | `66010101785` | -                                |
+| `0900000036112` | yes (Gent)            |     9090    |           no             |    active    | `08091860684` | -                                |
+| `0900000002312` | yes (Gent)            |     9000    |           no             |    blocked   | `71030360442` | -                                |
+| `0900000058918` | expired               |     1000    |           no             |    active    | `17010180147` | -                                |
+| `1000000520006` | no                    |     9000    |           no             |    active    | `80010101392` | -                                |
+| `0900000093204` | yes (in other region) |     9000    | 2 coupons (other region) |    active    | `10010105762` | -                                |
+| `0900000074519` | yes                   |  n/a        |           n/a            | unregistered | N/A           | -                                |
+| `0900000075706` | no                    |  n/a        |           n/a            | unregistered | N/A           | -                                |
 
 ## Group passes
 
-| UitpasNumber      | Social tariff | Coupons | Card status |
-|----------|:-------------:|:-------------:|:-------------:|
-| `0900000045410` | yes | no | active |
+| UitpasNumber    | Social tariff | Coupons | Card status | Comment                     |
+|---------- ------|:-------------:| :-----: | :---------: | :-------------------------: |
```

Comment:
I think the extra space here breaks the table. See https://docs.publiq.be/docs/uitpas/branches/UPS-add-testdata-details/238aeb19e10c1-ui-tpas-test-dataset

```suggestion
|----------------|:-------------:| :-----: | :---------: | :-------------------------: |
```

Reply:
Note: The linter should also format the tables if you run that (but only if the table has a valid syntax which is now not the case anymore) https://github.com/cultuurnet/apidocs#automatically-fixing-some-errors-